packages/frontend/src/components/send/components/InputAccountId.js

```diff
@@ -2,6 +2,8 @@ import React, { Component, createRef } from 'react';
 import { Translate } from 'react-localize-redux';
```

Comment:
InputAccountId should know nothing about HAPI checks.
Can we decouple logic from there?

Reply:
moved it up two levels